Sources/Citadel/TTY/Client/TTY.swift

```diff
@@ -304,4 +373,47 @@ extension SSHClient {
         
         return ExecCommandStream(stdout: stdout, stderr: stderr)
     }
+    
+    /// Requests a shell to be invoked and executes a command on the remote server.
+    /// - Parameters:
+    /// - command: The command to execute.
+    public func executeInShell(command: String) async throws -> AsyncThrowingStream<ByteBuffer, Error> {
+        var streamContinuation: AsyncThrowingStream<ByteBuffer, Error>.Continuation!
```

Comment:
I know I do this a fair bit, but I don't think we need to have a force unwrap here, if things are re-ordered a little. Ultimately, it's needed to create a handler, and we only need the handler down where we create the channel and configure its pipeline. So if we can create the stream in that location, that would be cleaner.

Reply:
This is also copy pasted. Maybe we can extract and reuse this in the other methods as well?
Are you suggesting doing everything in the `AsyncThrowingStream` init closure?

Reply:
Not everything per-say, but I do think it can help us get rid of the implicitly unwrapped optional